### PR TITLE
master-qa-51488_64_2

### PR DESCRIPTION
**Pull request by brianwulbern**
- LRQA-51488 Align string with Pages behavior
- LRQA-51488 Regen

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
Thanks @brianwulbern!

**Comment by jkappler** [comment]
ci:forward

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -17966,7 +17966,7 @@ this-page-is-linked-to-a-site-template-which-does-not-allow-modifications-to-it=
 this-page-is-not-a-content-display-page-template=This page is not a content display page template. It is possible to make an existing page a display page by adding an Asset Publisher and configuring it to be the default Asset Publisher for the page.
 this-page-is-not-enabled-in-this-site-pages-variation,-but-is-available-in-other-variations=This page is not enabled in this Site Pages Variation, but is available for other pages variations.
 this-page-is-part-of-an-inactive-portlet=This page is part of an inactive portlet.
-this-page-is-using-a-different-theme-than-the-one-set-for-pages=This page is using a different theme than the one set for pages. Note that style books will only be available for pages that use the theme set in pages.
+this-page-is-using-a-different-theme-than-the-one-set-for-pages=This page is using a different theme than the one set for all pages. Note that style books will only be available for pages that use the theme set in all pages.
```
Comment (victorg1991) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:17969]: hey @brianwulbern could you update also the key. I believe it should be `this-page-is-using-a-different-theme-than-the-one-set-for-all-pages`

Comment (brianwulbern) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:17969]: Ok

**Comment by brianwulbern** [comment]
@victorg1991 @jkappler 

Resent to https://github.com/liferay-echo/liferay-portal/pull/11690

**Pull request closed without merging**